internal/controllers/comments/course_group.go

```diff
@@ -37,8 +39,23 @@ func CourseGroupComment(c *fiber.Ctx) (err error) {
 	}
 	var response []CommentResponse
 	response = GenerateResponse(comments, uid, likeResult, true, utils.GetIP(c))
-	return c.Status(fiber.StatusOK).JSON(models.OKResponse{
-		Data:  response,
-		Error: false,
-	})
+	currentUser, err := queries.GetUserByID(nil, uid)
+	if err != nil {
+		return err
+	}
+	if currentUser.IsAdmin || currentUser.IsCommunityAdmin {
+		return c.Status(fiber.StatusOK).JSON(models.OKResponse{
+			Data:  response,
+			Error: false,
+		})
+	} else {
+		for i := range response {
+			// 设置评论的 Reward 字段为 -1，表示不可见
+			response[i].Reward = -1
+		}
+		return c.Status(fiber.StatusOK).JSON(models.OKResponse{
+			Data:  response,
+			Error: false,
+		})
+	}
```

Comment:
Extract common statement. Avoid duplicate code.

```suggestion
	if !currentUser.IsAdmin && !currentUser.IsCommunityAdmin {
		for i := range response {
			// 设置评论的 Reward 字段为 -1，表示不可见
			response[i].Reward = -1
		}
	}
	return c.Status(fiber.StatusOK).JSON(models.OKResponse{
		Data:  response,
		Error: false,
	})
```